examples/fastspeech2_multispeaker/README.md

```diff
@@ -0,0 +1,23 @@
+# Fast speech 2 multi-speaker
+
+## Prepare
+Everything is done from main repo folder so TensorflowTTS/
+
+0. Optional* Download and prepare libritts (helper to prepare libri in examples/fastspeech2_multispeaker/libri_experiment/prepare_libri.ipynb)
```

Comment:
add link download, the prepare_libri notebook has some bugs :)). `{libri_path}/SPEAKERS.txt` will failed if `libri_path` = "LibriTTS/" then there is no "LibriTTS//SPEAKERS.txt". There are some typos also (change `daataset_path` -> 'dataset_path'). Also, there are some variable should be describe:

max_speakers = 20
min_len = 20
max_file_len = 11
min_file_len = 2.

The code `to_move = to_move[:20]` is so specific in ur case since you just training the small subset. Let add note for this code.

Reply:
its a bug will fix it :P